app/ui/app/src/components/StreamingMarkdownContent.tsx

```diff
@@ -1,132 +1,146 @@
 import React from "react";
```

Comment:
Scrolling thinking doesn't seem to work on this PR, I'm not sure what it's from though